app/otr/client/rocketchat.otr.room.js

```diff
@@ -67,6 +68,7 @@ OTR.Room = class {
 	}
 
 	end() {
+		APIClient.v1.post('statistics.telemetry', [{ eventName: 'otrEnded', timestamp: Date.now(), roomId: this.roomId }]);
```

Comment:
```suggestion
		APIClient.v1.post('statistics.telemetry', [{ eventName: 'otrEnded', timestamp: Date.now(), rid: this.roomId }]);
```

The type is set as `rid` and not roomId